src/components/Button/Button.tsx

```diff
@@ -198,8 +207,29 @@ export function Button({
   const { Link } = useComponents();
   const LinkButton = BaseButton.withComponent(Link);
   const ButtonElement = props.href ? LinkButton : BaseButton;
+
+  /** Set default value for tracking dispatch */
+  const { label = undefined, component = 'button', data = undefined } =
+    props.trackingPayload || {};
+  const hasTracking = !!label || props.enableTracking;
```

Comment:
I assume `enableTracking` is off by default? Why don't we turn it on by default?